README.md

```diff
@@ -3,136 +3,29 @@
-
-
-#### Quick start example
-
-```go
-// Generate Makefile.mta file.
-mbt init 
+#### Multi-Target Application
 
-// Execute MTA project build.
-make -f Makefile.mta p=cf
+Before using this package, be sure you are familiar with the multi-target application concept and terminology. For background and detailed information, see the [Multi-Target Application Model](https://www.sap.com/documents/2016/06/e2f618e4-757c-0010-82c7-eda71af511fa.html) guide. 
 
-```
-
-
-### Milestone 1  - (Q1-Q2 - 2019)
-
- - [x] Supports project-assembly-based deployment descriptors.
- - [x] Supports the building of HTML5 applications (non repo).
- - [x] Supports the building of node applications.
- - [ ] Partially supports build parameters (first phase):
-    - [ ] Supports build dependencies.
-    - [ ] Supports the copying of build results from other modules.
-    - [ ] Supports the build results from a different location.
-    - [x] Supports target platforms.
- - [x] Supports the generation of a default `Makefile` file.
- - [ ] Supports the generation of an `mtad.yaml` file from an `mta.yaml` file.
- - [ ] Supports the building of `XSA` and `CF` (Cloud Foundry) targets.
-
-### Milestone 2 - (Q2 - 2019)
-
-  - [ ] Supports the generation of verbose `Makefile` files.
-  - [ ] Supports MTA extensions.
-  - [ ] Supports the building of Java and Maven applications.
-  - [ ] Supports ZIP builds.
-  - [ ] Supports fetcher builds.
-  - [ ] Supports build parameters:
-    - [ ] Supports build options.
-    - [ ] Supports `ignore` files and folders.
-    - [ ] Supports the definition of timeouts.
-    - [ ] Supports the naming of build artifacts.
-  - [ ] Supports multi-schema.
-  - [ ] Supports the enhancing of schema validations.
-  - [ ] Supports semantic validations.
-  - [ ] Partially supports the advanced `mta.yaml` (3.1 > 3.2) schema.
-
-
- ### Milestone 3 - (Q3 - 2019)
-
-  - [ ] Supports parallel execution for the default `Makefile` file.
-  - [ ] Supports incremental builds; in other words, one module at a time.
-
- ### Milestone 4 - (Q3 - 2019)
-
- - [ ] Supports the extensibility framework.
- - [ ] Fully supports the advanced `mta.yaml` (3.1 > 3.2) schema.
-
-## Download and Installation
-
- There are two supported ways to install the multi-target application archive builder (MBT) tool:
+#### The Cloud MTA Build Tool overview
+The Cloud MTA Build Tool is a standalone command-line tool that builds a deployment-ready
+multi-target application (MTA) archive `.mtar` file from the artifacts of an MTA project according to the project’s MTA
+development descriptor (`mta.yaml` file) or from module build artifacts according to the MTA deployment descriptor (`mtad.yaml` file). Also it provides commands for runing intermediate build process steps, for example, the mta.yaml file validations,building a single module according to configurations in the development descriptor, generating deployment descriptor, etc.
```

Comment:
Also, it provides command for running intermediate build process steps; for example, the "mta.yaml" file validations, building a single module according to the configurations in the development descriptor, ....

Reply:
Updated

Reply:
Thanks